lib/Runtime/Base/WindowsGlobalizationAdapter.cpp

```diff
@@ -385,7 +385,6 @@ namespace Js
 
         // Construct HSTRING of timeZoneId passed
         IfFailThrowHr(GetWindowsGlobalizationLibrary(scriptContext)->WindowsCreateStringReference(timeZoneId, static_cast<UINT32>(wcslen(timeZoneId)), &timeZoneHeader, &timeZone));
-        Assert(timeZone);
```

Comment:
why not return here? Also in this file many places we were asserting on this HSTRING for being there, could we just remove them?